.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "husky"]
+	path = husky
+	url = https://github.com/husky-team/husky.git
```

Comment:
Does Husky has a stable version? Not frequently modified except for some necessary bugfixes.

I have a suggestion. As more and more applications use Husky as a submodule, pyhusky Yuzhen's ml system(it uses Husky as a submodule, too), etc. It is necessary to provide a stable version of Husky as a submodule. If there is a big modification in stable Husky version, any project using it as a submodule can be updated at the same time. If not, any other time, the projects can run successfully. I think it will be good for pyhusky open. @ddmbr @kygx-legend @Yuzhen11